pajbot/bot.py

```diff
@@ -391,6 +391,12 @@ def get_date_value(self, key, extra={}):
         except:
             log.exception("Unhandled exception in get_date_value")
 
+    def get_datetimefromisoformat_value(self, key, extra={}):
+        try:
+            return datetime.datetime.fromisoformat(key).replace(tzinfo=datetime.timezone.utc)
```

Comment:
Should only replace the tzinfo with UTC if the parsed date has no tzinfo, since you can specify a timezone in ISO format strings as well (which we would discard, giving the wrong parsed datetime).
Maybe should also document that if no timezone is specified, UTC is assumed. (in variables.md)

Reply:
Should be all good in the last two commits :ok_hand: